File: operacoes.js
const assert = require('assert');

const soma = (a,b) => a + b
const sub = (a,b) => a - b
const divisao = (a,b) => a / b
const mult = (a,b) => a * b

try {
  assert.equal(10, soma(5,5));
  assert.equal(10, sub(20,10));
  assert.equal(10, divisao(20,2));
  assert.equal(10, mult(5,2));
} catch (e) {
  console.log(e);
}

File: C/operacoes.c
#include <stdio.h>
#include <assert.h>

int soma (int a, int b) 
{
  return a + b;
}
int sub (int a, int b) 
{
  return a - b;
}
int mult (int a, int b) 
{
  return a * b;
}
double divisao (double a, double b)
{
  return a / b;
}

int main() 
{
    assert(10 == soma(5,5));
    assert(2 == sub(5,3));
    assert(15 == mult(5,3));
    assert(6 == divisao(12,2));
    return 0;
}